notebooks/Image_Analysis.ipynb

```diff
@@ -236,41 +237,49 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from datasetinsights.stats.image_analysis import get_average_psd_1d"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "42400dbc",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "img_dir_path = \"<img-dir-path>\""
+    "psd_1d_dict = {}\n",
+    "for data in config:\n",
+    "    img_dir_path = config[data][\"img_dir_path\"]\n",
+    "    img_type = config[data][\"img_type\"]\n",
+    "    avg_psd_1d, _ = get_average_psd_1d(img_dir=img_dir_path, img_type=img_type)\n",
```

Comment:
the input path to this function is a bit different than ```img_dir_path```. Please update to:

```get_average_psd_1d(img_dir=img_dir_path+'/images/', img_type=img_type)```

Reply:
`img_dir_path` shoud be the absolute path of the directory where images are located. Why do you  need to add `images` to the path?